### PR TITLE
Update document processing

### PR DESCRIPTION
**Pull request by GJJ121785021**
Update document processing

### CODE REVIEW

**Review by abersheeran — DISMISSED** [review]
Good change！

**Review by waitan2018 — CHANGES_REQUESTED** [review]
class JustTest(View):
    def get(
        self,
        request: HttpRequest,
        id: int = Path(..., description="This is description of id."),
    ) -> HttpResponse:
        """This is summary.
        This is summary.

        This is description.
        This is description.

        This is description.
        This is description.
        """
        return HttpResponse(id)


Please make sure to pass the above test case！

File: django_simple_api/views.py [hunk 1]
```diff
@@ -23,13 +23,8 @@ def _generate_method_docs(function) -> Tuple[Dict[str, Any], Dict[str, Any]]:
...
     if isinstance(doc, str):
-        doc = doc.strip()
-        result.update(
-            {
-                "summary": doc.splitlines()[0],
-                "description": "\n".join(doc.splitlines()[1:]).strip(),
-            }
-        )
+        clean_doc = "\n".join(i.strip() for i in doc.strip().splitlines())
+        result.update(dict(zip(("summary", "description"), clean_doc.split("\n\n", 1))))
```
Comment (abersheeran) [inline django_simple_api/views.py:27]: Use `result.update(zip(...))` replace to `result.update(dict(zip(...)))`

Comment (GJJ121785021) [reply django_simple_api/views.py]: done!

**Review by waitan2018 — APPROVED** [review]
good job!

**Review by abersheeran — APPROVED** [review]

**Pull request merged**